test/hook.async.js

```diff
@@ -1,4 +1,4 @@
-describe('async', function(){
+describe.noRandom('async', function(){
```

Comment:
These tests must be run serially. These hunks make it possible to `mocha --randomize` on the Mocha source itself.